src/utils/getRpcUrl.ts

```diff
@@ -119,7 +119,6 @@ const fallBackProvider = (chainId: number) => {
 };
 
 /* Returns a PublicClient instance for the configured network. */
-// @ts-expect-error viem typings
```

Comment:
I noticed in the commit messages that this is supposed to not throw an error. It might have been that it was only doing that in Intellij editors. It might also be that this got fixed while upgrading typescript. The original message was about the nested types being too many levels deep, iirc.

Reply:
I removed it and my editor is fine and the build is working as expected. If you're still using Intellij, can you check if this is okay on your end?

Reply:
Also not seeing the type errors anymore, resolving this for now.